charts/kube-prometheus-stack/values.yaml

```diff
@@ -134,6 +135,13 @@ global:
   # or
   # - "image-pull-secret"
 
+windowsMonitoring:
+  ## Deploy the windows specific dashboards and rules, depends on deployment of windows_exporter
```

Comment:
Is there a reason not to include windows_exporter deployment files in this repo? Looks like the node_exporter deployment files are included.

Reply:
I would go like this for now but create a follow-up PR adding a separate chart for windows-exporter just like we have for node-exporter and when that one is merged we can add it as a dependency to kube-prometheus-stack in another PR.

Reply:
windows nodes are exotic. 
i would not add them to the chart by default.

Reply:
All resources added in this PR are disabled by default, so they don't affect anyone who doesn't explicitly opt in.
IMHO, Windows nodes were exotic in the past but nowadays, there are many people having Windows nodes in their production setup.

Reply:
>i would not add them to the chart by default.

Agree they shouldn't be deployed by default, but they could be behind a flag?